src/modules/universal-discovery/css/universal.discovery.module.css

```diff
@@ -12,11 +12,8 @@
     left: 50%;
     transform: translate(-50%, -50%);
     background: #eee;
-    padding: 16px;
     box-shadow: 0 0 30px 0 rgba(0,0,0,.35);
-    display: grid;
-    grid-template-columns: auto;
-    grid-template-rows: 48px 48px 1fr 48px;
+    border-radius: 6px;
```

Comment:
Please change: 
- `border-radius: 4px;`
- I would also suggest to change, at least, width to `width: 95vw;`, and also consider to change height to `height: 95vh;` in order to maximize UDW's features when in medium screen devices especially.